README.md

```diff
@@ -1,3 +1,33 @@
 # Website Development Environment
 
 Repository containing relevant Dockerfiles for creating a development environment for the CodeSupport website.
+
+## Summary
+
+This repo is aimed to increase the ease of development with the CodeSupport website stack. It bundles up both the backend and the frontend into two docker containers with the required dependencies to get the project running, while also cloning the projects locally and mounting them as volumes to ensure development is also seamless.
+
+## Getting started
+
+Firstly we need to create an OAuth2 application with discord [here](https://discordapp.com/developers/applications/#top) to be able to connect to the discord API. Once you have created a new application make note of the client ID and client secret.
```

Comment:
- A comma is needed after "Firstly".
- A capital letter is needed for "Discord".